src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './Footer.scss';
+import GoogleIcon from '../../../assets/google.png';
```

Comment:
@LukasChiama  is it not possible to have an index file for all these icon set

Reply:
I'll do that right away